### DISCUSSION TITLE
Does this consider Icons within the navigation?

### DISCUSSION THREAD

**Discussion by arcs-** [teamnovu/statamic-unused-assets#3] [Ideas]
I have a setup where users can add an Image to each navigation point but this plugin considers those as unused, have you noticed something like this before?

**Comment by kylewalow** [comment]
Yes, currently the 'Entries', 'Terms', and 'GlobalSets' are covered. Including 'Navigation' definitely makes sense since you can also add images to it.

**Reply by kylewalow** [reply]
Just created a Issue for it
